docs/uml/macros.iuml

```diff
@@ -312,7 +309,16 @@ skinparam defaultMonospacedFontName Andale Mono
     !endif
 !endif
 
+!ifdef SHOW_NETWORK_GENEALOGY
+    resource(NetworkGenealogy) {
+      + ancestor: Network
+      + descendant: Network
```

Comment:
Wait I've been spelling this wrong this whole time??

Reply:
I mean it is a pretty nuanced difference. https://cat.wordpandit.com/descendant-descendent/

Reply:
I love nuanced differences!